### PR TITLE
Fix #2011

### LINKED ISSUE #2011: Improve archive
When listing years or months, show the count of posts.

Example:

* 2004 (25)
* 2005 (23)

Or

* January (4)
* February (8)

### PR DESCRIPTION
**Pull request by jjconti**

### CODE REVIEW

**Comment by Kwpolska** [comment]
what about _generate_posts_task for daily archives?

File: nikola/data/themes/base/templates/list.tmpl [hunk 1]
```diff
@@ -8,8 +8,11 @@
     </header>
     %if items:
     <ul class="postlist">
-    % for text, link in items:
+    % for text, link, count in items:
```
Comment (Kwpolska) [inline nikola/data/themes/base/templates/list.tmpl]: other lists might not contain the count thing, you should look into that (in `self.site.generic_post_list_renderer`?)

Comment (jjconti) [inline nikola/data/themes/base/templates/list.tmpl]: I looked for uses of list.tmpl

Comment (Kwpolska) [inline nikola/data/themes/base/templates/list.tmpl]: Okay, let’s merge this.

**Pull request merged**